internal/store/postgres/resource_repository_test.go

```diff
@@ -162,7 +162,7 @@ func (s *ResourceRepositoryTestSuite) TestFind() {
 				actualResult, actualError := s.repository.Find(tc.filters)
 
 				s.NoError(actualError)
-				if diff := cmp.Diff(tc.expectedResult, actualResult, cmpopts.EquateApproxTime(time.Nanosecond)); diff != "" {
+				if diff := cmp.Diff(tc.expectedResult, actualResult, cmpopts.EquateApproxTime(time.Microsecond)); diff != "" {
```

Comment:
so, is it postgres not storing the microsecond? or `cmpopts.EquateApproxTime(time.Microsecond)` is the way to ignore the nanosecond?

Reply:
@rahmatrhd so golang `time.Now` is within nano second precision and postgres uses micro second precision.
above is to ignore up to microsecond difference